scripts/ci/reborn_changed_coverage.py

```
@@ -1,204 +1,424 @@
+                    "missing_files": [],
+                    "uncovered": [],
+                    "uncovered_branches": [],
+                    "passed": True,
+                },
+            )
+            print("Changed coverage: no Reborn production lines added")
+            return 0
+        if not coverage.saw_branch_records:
+            raise GateError(
+                "merged lcov contains no BRDA records; branch instrumentation is missing"
+            )
```

Ruff (0.16.0)

[warning] Abstract `raise` to an inner function

(TRY301)